lib/rollbax/reporter/standard.ex

```diff
@@ -6,154 +6,165 @@ defmodule Rollbax.Reporter.Standard do
 
```

Comment:
So this is the module that:

- When the Rollbax reporter has been added
- Handles the event that comes in
- If it's an :error event
 - formats the exception nicely, depending on where the error came from
 - returns a `Rollbax Exception` struct